examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -71,6 +72,9 @@ class WebGPURenderer {
 
 		this._renderTarget = null;
 
+		this._antialias = !! parameters.antialias;
```

Comment:
In order to keep the amount of "private" properties small, I suggest to remove `_antialias` and `sampleCount`. Both informations are already available over the `parameters` property.

Reply:
That sounds good to me. I overlooked renderer keeps `parameters`.

Reply:
Thanks, updated.